DEPS

```diff
@@ -511,7 +511,10 @@ deps = {
   },
 
    # Get the SDK from https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core at the 'latest' tag
-   # Get the toolchain from https://chrome-infra-packages.appspot.com/p/fuchsia/clang at the 'goma' tag
+   # Get the toolchain from https://chrome-infra-packages.appspot.com/p/fuchsia/third_party/clang at the 'integration' tag
```

Comment:
How do these toolchains relate to the one pulled above starting on line 490? That one looks pinned to a different version, so just want to be sure we won't be using that one by mistake.

Reply:
These are specifically for Fuchsia. There have been issues (on Darwin platforms) that have prevented us from using the same version everywhere. But yes, ideally, there should be no duplication and a single toolchain must be used everywhere.